pkg/virt-controller/watch/migration.go

```diff
@@ -1107,6 +1150,13 @@ func (c *MigrationController) sync(key string, migration *virtv1.VirtualMachineI
 		return fmt.Errorf("vmi is inelgible for migration because another migration job is running")
 	}
 
+	if err = c.handleMigrationBackoff(key, vmi, migration, pod); err != nil {
+		warningMsg := fmt.Sprintf("backoff migrating vmi %s/%s", vmi.Namespace, vmi.Name)
+		c.recorder.Eventf(vmi, k8sv1.EventTypeWarning, err.Error(), warningMsg)
+		log.Log.Object(migration).Warning(warningMsg)
```

Comment:
Do you need this? I believe you handle this in `handleMigrationBackoff`

Reply:
Yes I agree, the one is more informative as well, I'll remove it.